t/10-tests_overview.t

```diff
@@ -225,6 +225,15 @@ subtest 'todo-flag on test overview' => sub {
     $schema->txn_rollback;
 };
 
+subtest 'comma-separated filtering of flavor in query param' => sub {
```

Comment:
I have the impression that this doesnt cover the poo expectations. Likely a UI test is needed to show that tht filter preserve the flavors.. or do I understand it wrong? 

this test it seems that validates basically that the controller can handle comma-separated values

Reply:
Indeed, I am working on more test coverage now :)